include/sound/hda_register.h

```diff
@@ -264,6 +264,8 @@ enum { SDI0, SDI1, SDI2, SDI3, SDO0, SDO1, SDO2, SDO3 };
 #define AZX_REG_ML_LOUTPAY		0x20
 #define AZX_REG_ML_LINPAY		0x30
 
+#define ML_LOSIDV_STREAM_MASK		0xFFFE
```

Comment:
add a comment that bit0 is reserved, with BIT(1) mapping to stream1

Reply:
Thanks!